benchmarks/oscilator_integrator_order/integrator_order_experiment.m

```diff
@@ -100,7 +101,7 @@
             problem_options.N_sim = N_sim;
             % generate new model with updated settings;
             model = oscilator(struct);
-            integrator = nosnoc.Integrator(model, problem_options, solver_options);
+            integrator = nosnoc.Integrator(model, problem_options, integrator_options);
             [t_grid, x_res, t_grid_full, x_res_full] = integrator.simulate();
             stats = integrator.stats;
```

Comment:
broken, as the method stats does not exist in integrator (and is rather part of solver which was not transfered accordingly)

Reply:
Currently getting stats only makes sense for the FESD plugin and can be done through `integrator.plugin.stats`. This is now done in this benchmark.